packages/eslint-plugin/src/rules/ban-types.ts

```diff
@@ -34,9 +34,9 @@ function stringifyNode(
 }
 
 function getCustomMessage(
-  bannedType: null | string | { message?: string; fixWith?: string },
+  bannedType: null | true | string | { message?: string; fixWith?: string },
 ): string {
-  if (bannedType == null) {
+  if (bannedType == null || bannedType === true) {
```

Comment:
[Testing] Should this come with a `ban-types.test.ts` change? I don't think this is already covered?

Reply:
Yeah this was accidentally allowed by the schema (it was set to `{type: 'boolean'}` instead of `{enum: [false]}`) - but the value wasn't handled by the logic or properly documented.
Will add a test